source/feature-guide/feature-deriving.rst

```diff
@@ -616,6 +441,159 @@ ZCL function like:
 ...results in a private multiple\_response variable where the
 "Frequently" and "Always" categories are selected.
 
+
+Categorical Logic
+~~~~~~~~~~~~~~~~~
+
+Crunch provides logical operations like most languages do, but with a
+few enhancements.
+
+First, Crunch logic is actually based on *three* values so you can
+properly handle missing data. Many programming languages do this,
+too: SQL implements ``NULL`` in addition to ``TRUE`` and ``FALSE``,
+and general languages often have a similar ``None`` or ``NA`` value.
+
+In Crunch, rather than having a "boolean" type with true/false/null
+values, we instead allow *any* categorical variable to participate
+in logic. Categories which have the ``selected: true`` flag are
+"selected". Categories which have the ``missing: true`` flag are
+"missing", and any categories that lack both are "other".
+Any categorical variable that has at least one of each can function
+as a "logical variable".
+
+How can you obtain a logical variable? A few ways:
+
+- You can define it yourself simply by setting the ``selected: true``
+  flag on one or more categories when your data is uploaded. You are
+  probably already doing this for ``missing: true``.
+- You can use the `select_categories function <#Multiple-Response-Views>`__
+  to derive new Multiple Response variables.
+- `Logic Functions <#Logic-Functions>`__ like ``==``, ``>``, ``in``,
+  ``or``, and ``not`` all return logical variables.
+- You can use the "filter builder" in the web UI to quickly create
+  derived variables using the logic functions.
+
+What can you do with logical variables? A few things:
+
+-  You can use them as filters. Any rows where the logical variable
+   is "selected" match the filter and are retained; any that are
+   "other" or "missing" do not match and are filtered out.
+-  You can reduce them to just three values with the ``as_selected``
+   function. This lets you:
+
+   +  Get "the right numbers" for Multiple Response analyses by
```

Comment:
I would change 'Get "the right numbers" for ...' to 'Calculate proportions and other statistics accurately for...'